include/flexflow/parallel_tensor.h

```diff
@@ -99,13 +99,19 @@ struct ParallelTensorShape {
 
 std::ostream& operator<<(std::ostream&, ParallelTensorShape const &);
 
+typedef std::vector<ParallelTensorShape> ParallelTensorShapes;
+
 }; // namespace FlexFlow
 
 namespace std {
   template <>
   struct hash<FlexFlow::ParallelTensorShape> {
     size_t operator()(FlexFlow::ParallelTensorShape const &) const;
   };
+  template <>
+  struct hash<FlexFlow::ParallelTensorShapes> {
+    size_t operator()(FlexFlow::ParallelTensorShapes const &) const;
```

Comment:
I'd rather not add a hash function for something as general as `std::vector<ParallelTensorShape>`. Would probably be best to either create a custom hash (my preferred solution) or to create a separate container class for the vector (my less-preferred solution)